includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -1781,6 +1773,51 @@ public function update_address( $order_id ) {
 		}
 	}
 
+	/**
+	 * Maybe capture instantly.
+	 *
+	 * @param $order_id
+	 *
+	 * @throws \SwedbankPay\Core\Exception
+	 */
+	private function maybe_capture_instantly( $order_id ) {
```

Comment:
Perhaps `try_capture_instantly` is a better name?

```suggestion
	private function try_capture_instantly( $order_id ) {
```

Reply:
@asbjornu WooCommerce developers use "maybe_" prefix for method with similar logic